src/main/java/com/microsoft/azure/search/samples/demo/Address.java

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.azure.search.samples.demo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class Address {
+    public static final String STREET_ADDRESS = "StreetAddress";
+    public static final String CITY = "City";
+    public static final String STATE = "State";
```

Comment:
@ruiqiding I suggested some changes to the schema since you wrote this. A new version will be published soon. I'm going to set up a GitHub repo under Azure-Samples just for the test schema and data. Once it's available, I'll send you a link.